python/tk_desktop/rpc.py

```diff
@@ -397,8 +455,14 @@ def call(self, name, *args, **kwargs):
                 raise
 
         # read the result
-        result = pickle.loads(self._connection.recv())
-        logger.debug("multi client got result '%s'" % result)
+        try:
+            result = pickle.loads(self._connection.recv())
+        except OSError:
+            # On Linux, an exception will be raised here instead.
```

Comment:
Using multiprocessing under Python 3 introduced some behaviour change, which you can see here.